shell/platform/tizen/tizen_input_method_context.cc

```diff
@@ -201,6 +217,34 @@ bool TizenInputMethodContext::HandleEvasEventKeyUp(Evas_Event_Key_Up* event) {
       reinterpret_cast<Ecore_IMF_Event*>(&imf_event));
 }
 
+bool TizenInputMethodContext::HandleNUIEventKey(const char* key,
+                                                const char* string,
+                                                uint32_t modifiers,
+                                                uint32_t scan_code,
+                                                bool is_down) {
+  if (ShouldIgnoreKey(key, true)) {
+    return false;
+  }
+
+  if (is_down) {
+    Ecore_IMF_Event_Key_Down imf_event =
+        NUIEventKeyToEcoreImfEvent<Ecore_IMF_Event_Key_Down>(
+            key, string, modifiers, scan_code);
+
```

Comment:
Let's minimize use of [vertical whitespaces](https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace).

Reply:
done?